mava/utils/checkpointing.py

```diff
@@ -24,7 +24,7 @@
 from jax.tree_util import tree_map
 from omegaconf import DictConfig, OmegaConf
 
-from mava.systems.ppo.types import HiddenStates, Params
+from mava.systems.anakin.ppo.types import HiddenStates, Params
```

Comment:
We definitely shouldn't be using a PPO type here, but I guess that is not the concern of this PR.